src/sql-parser/src/ast/defs/statement.rs

```diff
@@ -1800,6 +1800,60 @@ pub struct ClusterOption<T: AstInfo> {
 }
 impl_display_for_with_option!(ClusterOption);
 
+#[derive(Debug, Clone, PartialEq, Eq, PartialOrd, Ord, Hash)]
+pub enum ClusterAlterOptionName {
+    /// The `Wait` option.
+    Wait,
+}
+
+impl AstDisplay for ClusterAlterOptionName {
+    fn fmt<W: fmt::Write>(&self, f: &mut AstFormatter<W>) {
+        match self {
+            ClusterAlterOptionName::Wait => f.write_str("WAIT"),
+        }
+    }
+}
+
+impl WithOptionName for ClusterAlterOptionName {
+    /// # WARNING
+    ///
+    /// Whenever implementing this trait consider very carefully whether or not
+    /// this value could contain sensitive user data. If you're uncertain, err
+    /// on the conservative side and return `true`.
+    fn redact_value(&self) -> bool {
+        match self {
+            ClusterAlterOptionName::Wait => false,
+        }
+    }
+}
+
+#[derive(Debug, Clone, PartialEq, Eq, Hash, PartialOrd, Ord)]
+pub enum ClusterAlterOptionValue {
+    For(Option<Value>),
```

Comment:
why is this an `Option`?

Reply:
I was trying some things to get past an issue, this should not be an option, good callout!